TicTacToe/GameRulesHandler.cs

```diff
@@ -0,0 +1,59 @@
+using System.Linq;
+
+namespace TicTacToe
+{
+    public static class GameRulesHandler
```

Comment:
I don't think this should be a static class. Would you ever need to have a GameRulesHandler without a Game?